frontend/ui/widget/inputtext.lua

```diff
@@ -755,6 +755,18 @@ function InputText:rightChar()
     self.charpos, self.top_line_num = self.text_widget:getCharPos()
 end
 
+function InputText:startLine()
```

Comment:
`startLine()` sounds like it would start a new line :)
The existing function names are a bit free style ("leftChar()" is not a verb), but as we have `delToStartOfLine()`, `goToEnd()`, may be `goToStartOfLine()` and `goToEndOfLine()` would add more consistency :)

Reply:
Given the keyboard association, I might even venture going with something around Home/End to mimic what those keys actually usually do (which is precisely "jump to beginning/end of line ^^" ;).

Reply:
> something around Home/End

Sure, but we have already for start/end of text
https://github.com/koreader/koreader/blob/c05b1a4ee45fd5336de9568b5890103e13cb3232/frontend/ui/widget/inputtext.lua#L758
https://github.com/koreader/koreader/blob/c05b1a4ee45fd5336de9568b5890103e13cb3232/frontend/ui/widget/inputtext.lua#L762